### PR TITLE
moltengamepad-git: init at 2016-05-04

### PR DESCRIPTION
**Pull request by ebzzry**

### CODE REVIEW

File: pkgs/misc/drivers/moltengamepad/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    owner = "jgeumlek";
+    repo = "MoltenGamepad";
+    rev = "6656357964c22be97227fc5353b53c6ab1e69929";
+    sha256 = "05cpxfzxgm86kxx0a9f76bshjwpz9w1g8bn30ib1i5a3fv7bmirl";
+  };
+
+  buildInputs = [ libudev ];
+
+  buildPhase = ''
+    mkdir -p $out/bin
```
Comment (fpletz) [inline pkgs/misc/drivers/moltengamepad/default.nix]: Is this directory need for `make`? If not, I think this command would make more sense in the `installPhase`.

**Comment by ebzzry** [comment]
@fpletz Ah, yes. That is correct. Fixed. (∩_∩)

**Comment by fpletz** [comment]
Thanks!

Travis fail is unrelated and fixed f0f31a256f833f124d3af2433e46431e547a3fb2.

**Comment by ebzzry** [comment]
@fpletz thanks! (｡◝‿◜｡)

**Pull request merged**